### ISSUE TITLE
batch size of plain CNN models

### ISSUE THREAD

**Issue by jodumagpi** [MeioJane/CHR#17]
It is mentioned that CHR approach used only one sample to update the learnable parameters, similar to stochastic gradient descent. I am wondering if, during the experiments, you also trained the plain CNN models in the same way. Did you also use only one sample for training plain models (say ResNet50, DenseNet, etc.)? If not, what is the batch size? More generally, can you state the training set up for both the plain CNN and CHR method? (Learning rate, optimizer, batch size, number of epochs, etc.) It is not clear whether the default values in your code were the exact values used in training. Also, did you use pretrained weights for the backbones?
